docs/bucket-notifications/README.md

```diff
@@ -0,0 +1,435 @@
+[![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio server supports Amazon S3 compatible bucket event notification for following targets [AMQP](https://www.amqp.org/about/what), [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started.html) , [Redis](http://redis.io/documentation), [nats.io](http://nats.io/) and [PostgreSQL](https://www.postgresql.org).
+
+##  Prerequisites
+
+* Install and configure Minio Server from [here](http://docs.minio.io/docs/minio).
+* Install and configure Minio Client from [here](https://docs.minio.io/docs/minio-client-quickstart-guide).
+
+| Notification Targets| 
+|:--
+| [`RabbitMQ`](#RabbitMQ) |   
+| [`Elasticsearch`](#Elasticsearch) |    
+| [`Redis`](#Redis) |    
+| [`NATS`](#NATS) |    
+| [`PostgreSQL`](#PostgreSQL) |    
+
+<a name="RabbitMQ"></a>
+## Publish Minio events via RabbitMQ 
+
+Install RabbitMQ from [here](https://www.rabbitmq.com/).
+
+### Recipe steps
+
+### Step 1: Add RabbitMQ endpoint to Minio
+
+The default location of Minio server configuration file is ``~/.minio/config.json``. Update the RabbitMQ configuration block in ``config.json`` as follows:
+
```

Comment:
It is a saperate note we have added, makes sense to be kept in bottom. 

That way someone will not get confused with distributed while the example workflow explaining stand alone. 

Please advice.